apps/src/applab/ApplabVisualizationColumn.jsx

```diff
@@ -37,6 +37,30 @@ class ApplabVisualizationColumn extends React.Component {
     widgetMode: PropTypes.bool
   };
 
+  componentDidMount() {
+    this.visualizationColumn.addEventListener(
+      'touchmove',
+      this.preventBehavior,
+      {
+        passive: false
+      }
+    );
+  }
+
+  componentWillUnmount() {
+    this.visualizationColumn.removeEventListener(
+      'touchmove',
+      this.preventBehavior,
+      {
+        passive: false
+      }
```

Comment:
i learned something interesting about this parameter and figured i'd share in case you haven't already seen it... i don't think this parameter is doing anything when used with `removeEventListener`. MDN has more info about it here: https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener#matching_event_listeners_for_removal

however, at the end of that section, they state:

> unless you have specific reasons otherwise, it's probably wise to use the same values used for the call to addEventListener() when calling removeEventListener()

Reply:
Interesting - thanks for sharing!